blog/_posts/2019-06-27-commit-of-the-month-june.md

```diff
@@ -0,0 +1,38 @@
+---
+slug: june-tilt-commit-of-the-month
+date: 2019-07-03T15:26:34.743Z
+author: dmiller
+layout: blog
+title: "Tilt Commit of the Month"
+subtitle: "June 2019 Edition"
+images:
+  - featuredImage.jpg
+image_type: "contain"
+tags:
+  - docker
+  - kubernetes
+  - microservices
+  - tilt
+  - commit
+keywords:
+  - docker
+  - kubernetes
+  - microservices
+  - tilt
+---
+
+Welcome to a new series of blog posts we're calling Tilt Commit of the Month. Commit of the Month is a lightweight way to highlight work that goes on in the Tilt project that might fly under the radar otherwise.
+
+We’re already cheating: this month we’re going to highlight a bunch of commits related to better detecting and working better with different kinds of Kubernetes clusters.
```

Comment:
the cheating bit is that it's MULTIPLE commits, right? that doesn't land -- consider rewording, using italics, etc.